File: app-mvp/app/src/main/java/com/bubbble/coreui/ui/adapters/viewholders/CommentViewHolder.java
package com.bubbble.coreui.ui.adapters.viewholders;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

import com.bubbble.R;
import com.bubbble.coreui.ui.views.dribbbletextview.DribbbleTextView;

public class CommentViewHolder extends RecyclerView.ViewHolder {

    public final TextView userName;
    public final ImageView userAvatar;
    public final DribbbleTextView commentText;
    public final TextView likesCount;

    public CommentViewHolder(View itemView) {
        super(itemView);

        userName = itemView.findViewById(R.id.user_name);
        userAvatar = itemView.findViewById(R.id.user_avatar);
        commentText = itemView.findViewById(R.id.text);
        likesCount = itemView.findViewById(R.id.likes_count);
    }

}

File: app-mvp/feature-shot-details/src/main/java/com/bubbble/shotdetails/comments/CommentViewHolder.kt
package com.bubbble.shotdetails.comments

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.bubbble.coreui.ui.views.dribbbletextview.DribbbleTextView
import com.bubbble.shotdetails.R

internal class CommentViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    val userName: TextView = itemView.findViewById(R.id.user_name)
    val userAvatar: ImageView = itemView.findViewById(R.id.user_avatar)
    val commentText: DribbbleTextView = itemView.findViewById(R.id.text)
    val likesCount: TextView = itemView.findViewById(R.id.likes_count)

}